paymaster/src/abis.ts

```diff
@@ -0,0 +1,29 @@
+import * as fs from 'fs';
+
+function readFile(path: string) {
+    try {
+        // Read the JSON file synchronously and store its contents in a variable
+        const jsonData = fs.readFileSync(path, 'utf8');
+  
+        // Parse the JSON data into an object
+        const jsonObject = JSON.parse(jsonData);
+
+        return jsonObject["abi"]
+    } catch (error) {
+        console.error('Error reading/parsing JSON file:', error);
+    }
+}
```

Comment:
Why not reuse the `read_json_file` from `libroll.py` here?

Reply:
This file is in Typescript.

Reply:
omg haha